src/less/breadcrumbs/ds6/breadcrumbs.less

```diff
@@ -7,4 +7,14 @@
 @breadcrumbs-item-focus-hover-color: @color-link-default;
 @breadcrumbs-item-current-color: @color-grey5;
 
+// CSS Custom Properties for DS6 only
+
+@media (prefers-color-scheme: dark) {
+    .switch {
+        --breadcrumbs-item-color: @color-text-default-dark-mode;
+        // --breadcrumbs-item-current-color: TBD for dark mode
+        --breadcrumbs-item-focus-hover-color: @color-link-default-dark-node;
```

Comment:
Typo: `dark-node` :-)

Reply:
~~You did not see that typo~~

Reply:
I did not see that typo

![movealng](https://user-images.githubusercontent.com/38065/81994577-03b9e380-95fd-11ea-85b7-e60f0a093779.jpg)